server/plugins/client/index.js

```diff
@@ -30,29 +53,9 @@ function register (server, options, next) {
             if (error) {
               return reject(error)
             }
+            decideAction(hasUpdate, bundleBuffer)
```

Comment:
I’m not sure about the naming of the function. Naming is hard :)

What it does is: it checks a few things and eventually writes the bundle to the file system. Maybe let’s call it `writeClientBundle(hasUpdate, inMemory, bundleTargetPath, bundleBuffer)`?